log/log.go

```diff
@@ -2,19 +2,14 @@ package log
 
 import (
 	"github.com/go-logr/logr"
-	"github.com/go-logr/zapr"
-	"go.uber.org/zap"
+
+	"k8s.io/klog/klogr"
 )
 
 // Logger is the root logger which should be used by all
 // other packages in the codebase.
 var Logger logr.Logger
 
 func init() {
-	// Set up logging.
-	zapLogger, err := zap.NewDevelopment(zap.AddCallerSkip(1), zap.AddStacktrace(zap.FatalLevel))
-	if err != nil {
-		panic(err)
-	}
-	Logger = zapr.NewLogger(zapLogger).WithName("router")
+	Logger = klogr.New()
```

Comment:
Do we need the `init()`? Can't this be:

    var Logger logr.Logger = klogr.New()

In fact, do we need the package? Can this just move to `main()`?

Reply:
> In fact, do we need the package? Can this just move to `main()`?

Oh, unless you want this as `log.Logger` - which does read a lot better.

Reply:
init can be removed, but the package is because otherwise all the library code would have to depend on the command package.

Reply:
Fixed